requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ pandas>=1.5,<2
 matplotlib>=3.6.0,<4
 numpy>=1.23.3,<2
 opensearch-py>=2
-torch<1.13.0
+torch<1.14.0
```

Comment:
Should we add `torch>1.13.0`?

Reply:
From https://github.com/opensearch-project/opensearch-py-ml/pull/56/checks?check_run_id=10183560962
> The Mend Security Check only scans your repository after a valid commit was pushed.

I'm ok to merge first to trigger security check if not sure if need to add `torch>1.13.0` or not

Reply:
I've already solved this in my PR: https://github.com/opensearch-project/opensearch-py-ml/pull/54

I don't think this will show up if my PR get merged.